src/bidiMapper/domains/script/scriptEvaluator.ts

```diff
@@ -316,180 +388,175 @@ export async function stringifyObject(
   return stringifyResult.result.value;
 }
 
-export class ScriptEvaluator {
```

Comment:
I needed to make one method global within this file, that's why I eliminated the class. It was outside of the scope where I needed to call it otherwise.

Reply:
This is not applicable anymore given the recent changes. Resolving.